test/token.test.js

```diff
@@ -338,7 +305,7 @@ describe('Token', () => {
         core.token.transfer(accounts[0], {
           from: safeAddresses[0],
           to: safeAddresses[4],
-          value: web3.utils.toBN(core.utils.toFreckles('5')),
+          value: web3.utils.toBN(core.utils.toFreckles('100000000')),
```

Comment:
It didn't pass with 5 freckles? 
This test is made for testing a transfer with a quantity (5) which fails at the beginning but after updating, succeeds.
If it doesn't work with a 5, we have to review the test more in detail.